src/video/renderers/opengl3/gl3_renderer.c

```diff
@@ -76,8 +78,8 @@ static bool setup_context(void *userdata, int window_w, int window_h, bool fulls
     if(!create_gl_context(&ctx->gl_context, ctx->window)) {
         goto error_1;
     }
-    if(!set_vsync(ctx->vsync)) {
-        goto error_2;
+    if((ctx->fake_vsync = !set_vsync(ctx->vsync)) == true) {
```

Comment:
```suggestion
    ctx->fake_vsync = !set_vsync(ctx->vsync);
    if(ctx->fake_vsync) {
```
Let's avoid assignment in conditionals unless there's a strong motivation.
Also, please remove useless boolean == true comparisons.

Reply:
The == true check makes it clear that it is an assignment and a comparison. I like this better stylistically. Faster to read.